### ISSUE TITLE
demo other types of links

### ISSUE THREAD

**Issue by johnmarriott** [BSteffaniak/MikeMcDannaldLab#4]
demo other types of links on the publications page: link to this article in the journal it was published, and others

e.g. for one paper:
- its page in the journal: http://learnmem.cshlp.org/content/23/11/618.abstract
- its page on researchgate: https://www.researchgate.net/publication/310823682_Subsecond_fear_discrimination_in_rats_Adult_impairment_in_adolescent_heavy_alcohol_drinkers

make the links an arbitrary collection - display names and uris

**Comment by BSteffaniak** [comment]
Should it be like 
`DiLeo A, Wright KM and McDannald MA (2016). "Sub-second fear discrimination in rats: Adult impairment in adolescent heavy alcohol drinkers." Learning & Memory 23: 618-622. [pdf] [journal] [ResearchGate]`
What should be the link text?

**Comment by johnmarriott** [comment]
Yes, that is a good template.  The link text can be specified in the array, like

`links: [ { title: "pdf", link: "/some/file.pdf" }, { title: "Learning & Memory", link: "http://journal.website/some/file.pdf}, ... ]`
